packages/contracts/contracts/SphinxBalanceFactory.sol

```diff
@@ -17,7 +17,9 @@ contract SphinxBalanceFactory {
         string indexed orgIdHash,
         address owner,
         string orgId,
-        address caller
+        address caller,
+        address balance,
+        address escrow
```

Comment:
Including the addresses here is just convenient since importing the necessary functions to calculate these addresses in the website backend was causing issues for me. I think I need to reorganize the project structure such that the website is a git module inside the mono repo, but it's not a high priority right now so I just fixed it by doing this. Also I think this is generally good info to include on the event anyway.